src/chains/ethereum_1/ethAction.ts

```diff
@@ -85,8 +84,13 @@ export class EthereumActionHelper {
       contract,
     } = actionInput
 
+    const gasPriceInWei =
```

Comment:
Add a clear comment as to types accepted by gasPrice
    // gasPrice is in GWEI - however when encoded to a HexString, it must be in WEI